### PR TITLE
Stylize webpack references

### PR DESCRIPTION
**Pull request by langri-sha**
It is officially written as __webpack__.

See https://twitter.com/wsokra/status/855800490713649152.

## What I did

Replaced all __Webpack__ occurrences with __webpack__, except when these appeared in titles.

### CODE REVIEW

File: docs/src/pages/configurations/custom-webpack-config/index.md [hunk 1]
```diff
@@ -5,9 +5,8 @@ title: 'Custom Webpack Config'
 
 ## Default mode
 
-The default Webpack config of Storybook is balanced for a medium-size project (specially created with [Create React App](https://github.com/facebookincubator/create-react-app)) or a library. But if you already have your own Webpack setup, that's not useable.
```
Comment (danielduan) [inline docs/src/pages/configurations/custom-webpack-config/index.md]: I know the existing doesn't make much sense anymore but we should still keep some of the same ideas presented here.

Something about the default being a sensible starting point for most projects and frameworks.

Comment (ndelangen) [reply docs/src/pages/configurations/custom-webpack-config/index.md]: agreed

Comment (ndelangen) [reply docs/src/pages/configurations/custom-webpack-config/index.md]: @langri-sha could you re-add something along these lines, do you think?

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
@danielduan I'll visit this line again later, I have no great replacement, but maybe in the future we'll find a good line.

Meanwhile I'm going to merge this PR, to move it along; it's been open way too long already.

**Pull request merged**